http_client.rst

```diff
@@ -686,6 +686,24 @@ You can either handle cookies yourself using the ``Cookie`` HTTP header or use
 the :doc:`BrowserKit component </components/browser_kit>` which provides this
 feature and integrates seamlessly with the HttpClient component.
 
+However, you're able to send cookies in your request. This is how you can do
+so using :class:`Symfony\\Contracts\\HttpClient\\HttpClient`::
+
+    use Symfony\Component\HttpClient\HttpClient;
+    use Symfony\Component\HttpFoundation\Cookie;
+
+    $client = HttpClient::create([
+        'headers' => [
+            'Cookie' => new Cookie('flavor', 'chocolate', strtotime('+1 day')),
```

Comment:
```suggestion
            'Cookie' => new Cookie('flavor', 'chocolate', strtotime('+1 day')),
            
            // or as plain string
            'Cookie' => USE_A_STRING_HERE
```

Reply:
Suits me, that's updated! :slightly_smiling_face: